mojave/items/guns/projectiles/rifle-proj.dm

```diff
@@ -40,6 +42,7 @@
 
 //5.56
 /obj/projectile/bullet/ms13/a556
+	icon_state = "bigroilfe_bullet"
```

Comment:
Could probably use a medium bullet maybe? 5.56 is not super strong.

Reply:
Yeah. I think the large bullets is what Infra intended to be used for stuff like AMRs and whatnot. 308s maybe. 5.56 if you REALLY wanted to be technical about it could probably even be portrayed by the small bullet. But medium will suffice for sake of consistency.

Reply:
I did it based on the file it was in

Reply:
merging these suggested changes in a second so resolving